doc/playbook.rst

```diff
@@ -214,44 +214,29 @@ The service can be controlled with ``kubectl`` from ``rubin-devl``.
-* IP_APDB: IP address or hostname and port of the APDB (see `Databases`_, below)
-* IP_REGISTRY: IP address or hostname and port of the registry database (see `Databases`_)
-* DB_APDB: PostgreSQL database name for the APDB
-* PSQL_APDB_PASS: secret containing the password for ``USER_APDB`` (see below)
-* DB_REGISTRY: PostgreSQL database name for the registry database
-* PSQL_REGISTRY_PASS: secret containing the password for ``USER_REGISTRY`` (see below)
-* KAFKA_CLUSTER: hostname and port of the Kafka provider
-
-The following environment variables are optional:
-
-* IMAGE_TIMEOUT: timeout in seconds to wait after expected script completion for raw image arrival, default 20 sec.
-* LOCAL_REPOS: absolute path (in the container) where local repos are created, default ``/tmp``.
-* USER_APDB: database user for the APDB, default "postgres"
-* USER_REGISTRY: database user for the registry database, default "postgres"
-* NAMESPACE_APDB: the database namespace for the APDB, defaults to the DB's default namespace
-* SERVICE_LOG_LEVELS: requested logging levels in the format of `Middleware's --log-level argument <https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/scripts/butler.html#cmdoption-butler-log-level>`_.
-  Default is to log prompt_prototype at DEBUG, other LSST code at INFO, and third-party code at WARNING.
-
-Secrets are configured through the makefile and ``kustomization.yaml``.
+The service container deployment is managed using `Argo CD and Phalanx <https://k8s.slac.stanford.edu/usdf-prompt-processing-dev/argo-cd>`_.
+See the `Phalanx`_ docs for information on working with Phalanx in general (including special developer environment setup).
+
+There are two different ways to deploy a development release of the service:
+
+* If you will not be making permanent changes to the Phalanx config, go to the Argo UI, select the specific ``prompt-proto-service-<instrument>`` service, then select the first "svc" node.
+  Scroll down to the live manifest, click "edit", then update the ``template.spec.containers.image`` key to point to the new service container (likely a ticket branch instead of ``latest``).
+  The service will immediately redeploy with the new image.
+  To force an update of the container, edit ``template.metadata.annotations.revision``.
+  *Do not* click "SYNC" on the main screen, as that will undo all your edits.
+* If you will be making permanent changes of any kind, the above procedure would force you to re-enter your changes with each update of the ``phalanx`` branch.
+  Instead, clone the `lsst-sqre/phalanx`_ repo and navigate to the ``applications/prompt-proto-service-<instrument>`` directory.
+  Edit ``values-usdfdev-prompt-processing.yaml`` to point to the new service container (likely a ticket branch instead of ``latest``) and push the branch.
+  You do not need to create a PR.
+  Then, in the Argo UI, select the service, click "SYNC" from the toolbar, and enter the branch name under "Revision".
```

Comment:
Might reference https://phalanx.lsst.io/developers/deploy-from-a-branch.html#switching-the-argo-cd-application-to-sync-the-branch for this

Reply:
Thanks for pointing that out! I didn't know about the "Target Revision" field. I wonder how permanent it is, compared to "Sync" → "Revision"?

Reply:
Looks to me that the  "Sync" → "Revision" is a one time syncing to the specified branch.  If that's different from the "Target Revision" field in the App Detail Summary, it'll show as "Out of sync" in its sync status.  If the "Target Revision" field is the ticket branch the sync status follows the ticket branch. 

Either sounds fine to me. In the end of the development work the target revision will be `main` either way.